R/twas.R

```diff
@@ -65,17 +65,20 @@ twas_z <- function(weights, z, R=NULL, X=NULL) {
 #' @importFrom foreach %dopar%
 #' @importFrom doParallel registerDoParallel
 #' @export
-twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, weight_methods = NULL, seed = NULL, num_threads = 1) {
+twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, prior_data_driven_matrices=NULL, weight_methods = NULL, seed = NULL, num_threads = 1) {
```

Comment:
I wonder if you can try to implement this interface:

`twas_weights_cv(X, Y, fold = NULL, sample_partitions = NULL, weight_methods = NULL, seed = NULL, num_threads = 1, ...)`


where `...` is left for other parameters specific to methods. Then you can match `...` with parameters such as `prior_data_driven_matrices`. for example, https://stackoverflow.com/questions/5890576/usage-of-three-dots-or-dot-dot-dot-in-functions

Reply:
Similarly this interface can also be added to `twas_weights` function.